JS Lab/Problem2.js

```diff
@@ -0,0 +1,31 @@
+(function(){
+    function Crate(weight, color){
+       this.weight = weight;
+       this.color = color;
+    }
+    var n = prompt("Enter an integer:");
+    var array = [n];
+    for(var i = 0; i < n; i++){
+        array[i] = [n];
+    }
+    var weight;
+    var color;
+    for(var i = 0; i < n; i++){
+        for(var j = 0; j < n; j++){
+            weight = Math.ceil(Math.random() * 9);
```

Comment:
Prefer using `const weight = <expression>` directly here instead of declaring `weight` before the loop starts.